app/serializers/encryptable_minimal_serializer.rb

```diff
@@ -1,5 +1,19 @@
 # frozen_string_literal: true
 
 class EncryptableMinimalSerializer < ApplicationSerializer
-  attributes :id, :name, :description
+  attributes :id, :name, :description, :used_attrs
```

Comment:
what does that do and where is it used?

Reply:
This is used in frontend in row component, to determine  what values are set in the encryptable, in the row component we only get the minimal data, and only when user reveals or copies the attribute, the encryptable is decrypted and sent again with the values. Because we now check how many attributes are used to display the correct amount of fields in the row, we need extra information from the backend.

This also can't just be a number of how many are used, because in that case we don't know what attributes are set and can't display them with the right values.